lib/pester.rb

```diff
@@ -56,12 +55,20 @@ def self.retry_action(opts = {}, &block)
         result = yield block
         return result
       rescue => e
-        class_reraise = opts[:retry_error_classes] && !opts[:retry_error_classes].include?(e.class)
+        retry_error_classes = opts[:retry_error_classes]
```

Comment:
i went with determination of retry instead of determination of reraise... seems a little more straightforward?